lib/rollbar.rb

```diff
@@ -363,6 +360,18 @@ def trace_data(exception)
       }
     end
 
+    def exception_backtrace(exception)
+      return exception.backtrace if exception.backtrace.respond_to?( :map )
+      return [] unless configuration.populate_empty_backtraces
+
+      cleaned_backtrace = caller.reject {|path| path =~ /#{rollbar_gem_dir}/ }
```

Comment:
Good point @brianr.

That's the issue, if we don't clean that filenames we will always show frames from the `Rollbar.error()` statement. But clean only those frames can be a little tricky.

Would you like to clean it in a smarter way? Not clean? Don't provide this feature?